### PR TITLE
Expose the instantiated network.

### PR DESCRIPTION
**Pull request by zoq**
Expose the the instantiated network; see https://github.com/mlpack/mlpack/issues/2041 for more details.

### CODE REVIEW

**Comment by zoq** [comment]
I like the ``LayerParameters`` idea, will open another PR.

File: src/mlpack/methods/ann/ffn.hpp [hunk 1]
```diff
@@ -265,7 +273,7 @@ class FFN
   //! Modify the initial point for the optimization.
   arma::mat& Parameters() { return parameter; }
 
-  //! Get the matrix of responses to the input data points.
+  //! Get the matrix of resposnses to the input data points.
```
[the remark this answers is not in the document]
Comment (zoq) [reply src/mlpack/methods/ann/ffn.hpp]: Opps, fixed in https://github.com/mlpack/mlpack/pull/2043/commits/dae940f24866d5bab3be3c72cfd9ef85b7cd8806.

**Comment by rcurtin** [comment]
Awesome, happy to have this merged. :+1:

**Comment by rirod17** [comment]
Thanks guys 👍

**Pull request merged**